src/megachatapi_impl.cpp

```diff
@@ -8010,7 +8010,12 @@ void MegaChatSessionHandler::onPeerMute(karere::AvFlags av, karere::AvFlags oldA
 
 void MegaChatSessionHandler::onVideoRecv()
 {
+    MegaChatCallPrivate *chatCall = callHandler->getMegaChatCall();
+    chatCall->sessionUpdated(session->peer(), session->peerClient(), MegaChatCall::CHANGE_TYPE_SESSION_OPERATIVE);
+    API_LOG_INFO("The call is fully operative. ChatId: %s, callid: %s",
+                 ID_CSTR(chatCall->getChatid()), ID_CSTR(chatCall->getId()));
```

Comment:
It would be good to log the `userid` and the `clientid`, since in groupcalls there will be more than one session, and they will receive more than one of this callbacks.

Reply:
I think that It could be useful to notify to each new session to know that a session i s fully operative.